src/frontend/next/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Footer from './Footer';
```

Comment:
What is the point of these files? Can the Footer component not exist without these?

Reply:
This was the folder structure in the gatsby front-end for all of the components made there. Iirc, we agreed to keep that structure for consistency. The reasoning for the files can be found [here](https://stackoverflow.com/questions/44092341/how-do-index-js-files-work-in-react-component-directories)

Reply:
Awesome, thanks!

Reply:
This can be a topic of discussion at your meetings because technically you don't need to have `index.tsx` in each of these components. You can simply:
get rid of  `index.tsx` and import like this `import Footer from '../components/Footer/Footer';`

or

keep `index.tsx` and import this way `import Footer from '../components/Footer;`